lib/archive/diff.go

```diff
@@ -84,28 +109,33 @@ func Tar(op trace.Operation, dir string, changes []docker.Change, spec *FilterSp
 				op.Errorf("Error closing tar writer: %s", cerr.Error())
 			}
 			if err == nil {
-				op.Debugf("Closing down tar writer with clean exit: %s", cerr)
+				if cerr != nil {
+					op.Errorf("Closing down tar writer with clean exit: %s", cerr)
+				} else {
+					op.Infof("Closing down tar writer with pristine exit")
```

Comment:
Yeah I think it should be, if we do not get an error then we likely should not log all of the time imo. The above situation is pretty much binary.